scripts/test_api_coverage.sh

```diff
@@ -1,5 +1,5 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I used `/bin/sh` before because I hope to limit scripts with POSIX shell for better compatibility and portability.

I know `bash` provides more features. We could migrate to `bash` since it's already widely provided across Linux distributions. If so, please change the 2nd line to `set -euo pipeline`, or even `set -euxo pipeline` for better debugging. (`sh` doesn't support `set -o pipeline`)

Reply:
I had to switch to Bash because the `tee >(go-junit-report >test-api-coverage-report.xml)` construction doesn't work correctly in SH
I'll add the `set -euxo pipeline`

Reply:
https://github.com/photoview/photoview/actions/runs/11998663626/job/33445679242?pr=1136#step:6:18 - `set -euxo pipeline` causes error. Please suggest how to fix it. For now I'll revert it back.

Reply:
Sorry, my fault. It should be `set -euxo pipefail`.

Reply:
Done